packages/cysync-core/src/dialogs/Inheritance/ExecutorMessage/context/index.tsx

```diff
@@ -0,0 +1,106 @@
+import React, {
+  Context,
+  FC,
+  ReactNode,
+  createContext,
+  useContext,
+  useMemo,
+} from 'react';
+
+import { ITabs, useTabsAndDialogs } from '~/hooks';
+
+import { closeDialog, useAppDispatch } from '~/store';
+import { FetchData, EditMessage, Success } from '../Dialogs';
+
+export interface InheritanceExecutorMessageDialogContextInterface {
```

Comment:
Update the name in all files and folders.

```suggestion
export interface InheritanceEditExecutorMessageDialogContextInterface {
```